test/transport/big.test.js

```diff
@@ -33,11 +33,11 @@ test('eight million lines', { skip }, async ({ equal, comment }) => {
   await pipeline(createReadStream(destination), split(), new Writable({
     write (chunk, enc, cb) {
       if (count % (toWrite / 10) === 0) {
-        comment(`read ${count}`)
+        t.diagnostic(`read ${count}`)
```

Comment:
Unfortunately, `t.diagnostic` does not work the same way as `tap.comment`. It only reports things at the end of a test run. We should just remove this `if` block.